turbot/table_turbot_notification.go

```diff
@@ -0,0 +1,609 @@
+package turbot
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strconv"
+	"time"
+
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+func tableTurbotNotification(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "turbot_notification",
+		Description: "Notifications from the Turbot CMDB.",
+		List: &plugin.ListConfig{
+			Hydrate: listNotification,
+			KeyColumns: plugin.KeyColumnSlice{
+				{Name: "id", Require: plugin.Optional},
+				{Name: "notification_type", Require: plugin.Optional},
+				{Name: "control_id", Require: plugin.Optional},
+				{Name: "control_type_id", Require: plugin.Optional},
+				{Name: "control_type_uri", Require: plugin.Optional},
+				{Name: "resource_id", Require: plugin.Optional},
+				{Name: "resource_type_id", Require: plugin.Optional},
+				{Name: "resource_type_uri", Require: plugin.Optional},
+				{Name: "policy_type_id", Require: plugin.Optional},
+				{Name: "policy_type_uri", Require: plugin.Optional},
+				{Name: "actor_identity_id", Require: plugin.Optional},
+				{Name: "create_timestamp", Require: plugin.Optional, Operators: []string{">", ">=", "=", "<", "<="}},
+				{Name: "filter", Require: plugin.Optional},
+			},
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
+			Hydrate:    getNotification,
+		},
+		Columns: []*plugin.Column{
+			// Top columns
+			{Name: "id", Type: proto.ColumnType_INT, Transform: transform.FromField("Turbot.ID"), Description: "Unique identifier of the notification."},
+			{Name: "notifications_count", Type: proto.ColumnType_INT, Transform: transform.FromField("Count"), Description: "Total number of notification matching the searched filter in query. It will be same for all the records in the table for a query."},
```

Comment:
Can you please remove this column? I think we'll add the ability to get counts but probably in a separate table.